dvc/logger.py

```diff
@@ -143,6 +143,7 @@ def _stack_trace(exc_info):
 
 
 def disable_other_loggers():
+    logging.captureWarnings(True)
```

Comment:
Oh, good to know there is such a feature! I now wonder If we still need https://github.com/iterative/dvc/blob/master/dvc/__init__.py#L18 .

Reply:
But also, these will still appear in regular output, because all warning do (and they should). So not sure how useful this change is.

Reply:
Ok, so it will redirect them to `py.warnings` logger, so we won't print it in CLI, so it means that indeed that ^ line where we handle warnings from numpy is no longer needed.